### DISCUSSION TITLE
If I want to create many telegram bots in one host

### DISCUSSION THREAD

**Discussion by AntonNosovich** [def-studio/telegraph#466] [Q&A]
Hello everyone, the problem is this: if I want to create several bots on one host, with different behavior, is there an example of how to do this?

**Comment by fabio-ivona** [comment]
Hi!

you can create as many bots as you like, inside the webookhandler, simply check the current bot and switch to its handling logic

**Reply by AntonNosovich** [reply]
Can you get me a example?
